modules/utxo-lib/src/bitgo/bitcoincash/address.ts

```diff
@@ -47,7 +47,7 @@ export function getPrefix(network: Network): string {
     case networks.ecash:
       return 'ecash';
     case networks.ecashTest:
-      return 'ecashtest';
+      return 'ectest';
```

Comment:
Expected statically returned data changed. Needs breaking change in commit footer.

Reply:
Commit msg amended + repushed